### ISSUE TITLE
Idea: community map as offered by OSM

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass-website#164]
Just came across this site:
https://openstreetmap.community/

Pretty nice idea as it shows you what's available in your region, for example Southern America:

![image](https://user-images.githubusercontent.com/1295172/83192551-fee54b80-a135-11ea-8aa6-b4af62c7628e.png)

Might be nice to do this for GRASS GIS as well.

**Comment by lucadelu** [comment]
> 
> Might be nice to do this for GRASS GIS as well.

I like the idea of the map but I don't think that "community map" is the more appropriate since we have not many communities as OSM, probably an user map could be more useful.

We could add a geojson file ad user can add them self doing PR (also editing the file online).

I can work on it if you like the idea

**Comment by veroandreo** [comment]
I really like the idea of having a user map in the website :)

**Comment by lucadelu** [comment]
I started to work on it. Following a preview

![community_map](https://user-images.githubusercontent.com/178741/83969983-4a84bb80-a8d3-11ea-9629-10e905eda37a.png)

I would like to improve the map with:
- [ ] a layer of companies/universities/etc contributing GRASS GIS
- [ ] show contributors to GRASS, this could be done with a new layer or with adding a new attribute field to user.geojson, do you have any preference?

**Comment by veroandreo** [comment]
will this be a separate page or a widget in the side panel of About/Community, for example?

**Comment by lucadelu** [comment]
separate page

**Comment by veroandreo** [comment]
One more thing. Given that CRS are so relevant for GRASS users, is it possible to use a different projection than that one? Greenland and Antarctica look so big and Africa so small... 

An equal area projection like Mollweide, Hammer, Goode homolosine, among others, would look much better in my opinion

**Comment by lucadelu** [comment]
I think your proposal does not fit properly with a web map. The used project is the [OpenLayers default one](https://openlayers.org/en/latest/doc/faq.html#what-projection-is-openlayers-using-), this is a projection developed by google specifically for web map.

However if we want to change projection we need to find a base layer with the chosen projection because OSM is provided only in Google projection.

**Comment by mlennert** [comment]
> I think your proposal does not fit properly with a web map. The used project is the [OpenLayers default one](https://openlayers.org/en/latest/doc/faq.html#what-projection-is-openlayers-using-), this is a projection developed by google specifically for web map.
> 
> However if we want to change projection we need to find a base layer with the chosen projection because OSM is provided only in Google projection.

Should be possible to reproject: https://openlayers.org/en/master/examples/reprojection.html.

**Comment by neteler** [comment]
> I think your proposal does not fit properly with a web map. The used project is the [OpenLayers default one](https://openlayers.org/en/latest/doc/faq.html#what-projection-is-openlayers-using-), this is a projection developed by google specifically for web map.

This doesn't necessarily make it a good projection ;-)
 
> However if we want to change projection we need to find a base layer with the chosen projection because OSM is provided only in Google projection.

Ok, I see the point. It might be something like
- https://openlayers.org/en/latest/examples/sphere-mollweide.html
- https://epsg.io/8857

Of course it's fine to address that in a separate PR.

**Comment by lucadelu** [comment]
> 
> Should be possible to reproject: https://openlayers.org/en/master/examples/reprojection.html.

yes, did you see the result when you choose Mollweide in the View projection?
For my point of view is really ugly

**Comment by veroandreo** [comment]
it looks like Mollweide 

![image](https://user-images.githubusercontent.com/20075188/84158786-8dc66200-aa6c-11ea-907a-a5aa704369cf.png)

**Comment by lucadelu** [comment]
> it looks like Mollweide
> 
> ![image](https://user-images.githubusercontent.com/20075188/84158786-8dc66200-aa6c-11ea-907a-a5aa704369cf.png)

in the central World it is fine, but in the corner no

![mollwaide](https://user-images.githubusercontent.com/178741/84159556-7cca2080-aa6d-11ea-9bdf-6e9a7bfac93f.png)

probably with a base layer provided in Mollweide projection the corners could be better

**Comment by mlennert** [comment]
> > Should be possible to reproject: https://openlayers.org/en/master/examples/reprojection.html.
> 
> yes, did you see the result when you choose Mollweide in the View projection?
> For my point of view is really ugly

;-) Question of aesthetics, I guess. But these aesthetics are unfortunately strongly influenced by the classroom Mercator maps we grew up with... 

Also watch out, on that page, you cannot zoom out to the entire world view. Does this look ugly to you, too:
[https://en.wikipedia.org/wiki/Mollweide_projection#/media/File:Mollweide_projection_SW.jpg](https://en.wikipedia.org/wiki/Mollweide_projection#/media/File:Mollweide_projection_SW.jpg)
?

You can go through some of the 'world' projections in QGIS and see what pleases you most, but IMHO it should be some equal area one.

**Comment by mlennert** [comment]
Here are some others (overlapping with some of Markus' suggestions):
[https://en.wikipedia.org/wiki/Equal_Earth_projection](https://en.wikipedia.org/wiki/Gall%E2%80%93Peters_projection)
[https://en.wikipedia.org/wiki/Gall%E2%80%93Peters_projection](https://en.wikipedia.org/wiki/Gall%E2%80%93Peters_projection)
[https://en.wikipedia.org/wiki/Strebe_1995_projection](https://en.wikipedia.org/wiki/Strebe_1995_projection)
[https://en.wikipedia.org/wiki/Eckert_II_projection](https://en.wikipedia.org/wiki/Eckert_II_projection)

but the earth being round, you will never have a beautiful rendering everywhere. And I don't think we have to many users that would register at the poles ;-)

**Comment by lucadelu** [comment]
Maybe the Gall-Peters could work since it is squared, need to test.

However my main concern is to have a beautiful map, not the most correct. We have plain of projection system and each system has it use, most of your proposal works really well for paper but less for web (for my point of view, and yes probably I'm influenced by Mercator too). If we find a beautiful tile server (or WMS service) of worldwide OSM data of one of above projection systems I will try with any of them ;-)

So, right now I will not work on this topic, please discuss in another ticket when community map will be merged in master
